xsd/siri_model/siri_estimatedVehicleJourney-v2.0.xsd

```diff
@@ -408,28 +395,33 @@ the original journey and the nature of the difference.</xsd:documentation>
    <xsd:element ref="Extensions" minOccurs="0"/>
   </xsd:sequence>
  </xsd:complexType>
+ <!-- ======================================================================= -->
  <xsd:element name="EstimatedCall" type="EstimatedCallStructure">
   <xsd:annotation>
    <xsd:documentation>Ordered sequence of SCHEDULED STOP POINTs called at by the VEHICLE JOURNEY. If IsCompleteStopSequence is false, may be just those stops that are altered.</xsd:documentation>
   </xsd:annotation>
  </xsd:element>
+ <xsd:element name="ExtraCall" type="xsd:boolean" default="false">
+  <xsd:annotation>
+   <xsd:documentation>This ESTIMATED CALL is an addition to the respective journey in the production timetable or to the previously sent prediction. If omitted: CALL is planned.</xsd:documentation>
+  </xsd:annotation>
+ </xsd:element>
+ <xsd:simpleType name="CallCancellation">
+  <xsd:annotation>
+   <xsd:documentation>This ESTIMATED CALL is a cancellation of a previously announced call.</xsd:documentation>
```

Comment:
Leave out ESTIMATED, because this type is also used in the RecordedCallStructure.